core/array/any_spec.rb

```diff
@@ -0,0 +1,30 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Array#any?" do
+  describe 'with no block given' do 
+    it "checks if there are any members of an array" do 
```

Comment:
Ah good find! Indeed for Hash#any? this detail does not matter since an Array is yielded and that is always truthy.